tests/full-example.ds

```diff
@@ -0,0 +1,42 @@
+extern {
+  fn format(x: str) : str;
+  fn printf(x: str);
```

Comment:
🤔 we need varargs support

Reply:
But somehow main.ds works out of the box...?